### PR TITLE
Adding no-hardware can/cannot do lists

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
@jacoffey3 I could use help with some of the wording and items. I feel like this is only 80% of the way there, but I'm not sure what's missing or needs to be tweaked

**Comment by CatGarab** [comment]
[What can you not do without a Board?](https://trello.com/c/evQFjZLi/1103-what-can-you-not-do-without-a-board)

File: docs/getting-started/no-board.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+What Can Be Done
+----------------
+
+- Tutorials - The majority of the KubOS :doc:`tutorials <../tutorials/index>` can be completed
+  without hardware. These tutorials will familiarize developers with basic system interactions and
+  architecture
+- Limited application development - High-level mission logic can be stubbed out and tested by running
+  everything within a local development environment. Once hardware is acquired, the stubbed out
+  code can be replaced with proper system calls.
+- Core service testing - All :doc:`core service <../ecosystem/services/core-services>` can be run
```
Comment (jacoffey3) [inline docs/getting-started/no-board.rst]: Is this actually true?

File: docs/getting-started/no-board.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+I Don't Have Hardware Yet, What Can I Do?
+=========================================
+
+Frequently mission projects are started with an idea of the intended mission architecture, but before
+hardware has been received.
+Hardware acquisition lead times can be quite long, in the realm of months, making it useful for
+mission engineers to be able to begin developing their system's software before their hardware
+arrives.
+
+The Kubos SDK allows some of the high-level mission logic to be fleshed out and tested before
```
Comment (jacoffey3) [inline docs/getting-started/no-board.rst]:
```suggestion
The Kubos SDK allows testing integration with :doc:`KubOS core services <../ecosystem/services/core-services>` and development of the high-level mission logic before
```

**Pull request merged**